css/css-values/calc-mixed-units.html

```diff
@@ -0,0 +1,31 @@
+<html lang="en">
```

Comment:
It's probably better to test in standards mode (i.e., with `<!DOCTYPE HTML>`) unless there's a good reason to test in quirks mode.